lib/SchemaPack.js

```diff
@@ -214,6 +217,47 @@ class SchemaPack {
     }
   }
 
+  /**
+   *
+   * @description Takes in a folder and identifies the related files from the
+   * root file perspective (using $ref property)
```

Comment:
Should we change using $ref property from description?

Reply:
changed